stage0/run.go

```diff
@@ -466,5 +468,5 @@ func launchMetadataService(debug bool) error {
 		Stderr: os.Stderr,
 	}
 
-	return cmd.Start()
+	return cmd.Run()
```

Comment:
@eyakubovich Yes, I thought we already had a discussion issue on the spec for that but can't seem to find it now. The more general point is we want to be more explicit about the utility of implementing the different parts of the spec independently